jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/view/category/category-detail-view.xml

```diff
@@ -0,0 +1,99 @@
+                <property name="categoryAttrs" fetchPlan="_local">
+                    <property name="defaultEntity" fetchPlan="_local"/>
+                    <property name="category" fetchPlan="_local"/>
+                </property>
+            </fetchPlan>
+            <loader id="categoryDl"/>
+            <collection id="categoryAttributesDc" property="categoryAttrs"/>
+        </instance>
+        <collection id="categoryAttributesLocationDc" class="io.jmix.dynattr.model.CategoryAttribute"/>
+    </data>
+    <facets>
+        <dataLoadCoordinator auto="true"/>
+    </facets>
+    <actions>
+        <action id="windowCommitAndClose" type="detail_saveClose"/>
+        <action id="windowClose" type="detail_close"/>
+    </actions>
+    <layout spacing="true" expand="tabSheet">
+        <tabSheet id="tabSheet" height="100%" width="100%">
+            <tab id="mainTab" label="msg://mainTab.title">
+                <vbox padding="false" margin="false">
+                    <formLayout id="form" dataContainer="categoryDc">
+                        <responsiveSteps>
+                            <responsiveStep minWidth="0" columns="1"/>
+                            <responsiveStep minWidth="28em" columns="2"/>
+                            <responsiveStep minWidth="42em" columns="3"/>
+                        </responsiveSteps>
+                        <textField id="nameField" property="name"
+                                   required="true" requiredMessage="msg://nameField.required"/>
+                        <comboBox id="entityTypeField"
+                                  label="msg://io.jmix.dynattr.model/Category.entityType"
+                                  required="true"
+                                  requiredMessage="msg://entityTypeField.required"/>
+                        <checkbox id="isDefaultField" property="isDefault"/>
+                    </formLayout>
+                    <vbox id="categoryAttrsBox" margin="false" padding="false">
+                        <h4 text="msg://io.jmix.dynattr.model/Category.categoryAttrs"/>
+                        <hbox id="buttonsPanel" classNames="buttons-panel">
+                            <button id="createBtn" action="categoryAttrsGrid.create"/>
+                            <button id="editBtn" action="categoryAttrsGrid.edit"/>
+                            <button id="removeBtn" action="categoryAttrsGrid.remove"/>
+                            <button id="moveUpBtn" action="categoryAttrsGrid.moveUp"/>
+                            <button id="moveDownBtn" action="categoryAttrsGrid.moveDown"/>
+                        </hbox>
+                        <dataGrid id="categoryAttrsGrid" dataContainer="categoryAttributesDc" width="100%" minHeight="20em"
+                                  height="100%">
+                            <actions>
+                                <action id="create" icon="PLUS" text="msg:///actions.Create" actionVariant="PRIMARY"/>
```

Comment:
You can use default `list_` actions and subscribe to `ActionPerformedEvent` to provide custom logic.

Reply:
I have to provide Category entity from parent view, so i cant do it

Reply:
You can. You subscribe to `ActionPerformedEvent` any way. If you subscribe to `ActionPerformedEvent` of some default action, you override its logic, but keeping visual appearance, like text, icon, variant.

Reply:
Okay, I misunderstood at first